src/styles/components/header.scss

```diff
@@ -1,2 +1,78 @@
 .header {
+    background: var(--color-background-darker);
+    font-size: 2rem;
+    font-weight: 600;
+    border-bottom: 1px solid #5d5c61;
```

Comment:
use rem here

Reply:
I assume you mean the border weight here, that has been updated now.